config/application.rb

```diff
@@ -96,6 +96,17 @@ class Application < Rails::Application
 
     config.active_record.verbose_query_logs = Settings.rails.active_record.verbose_query_logs
 
+    if Settings.exception_notification.enabled
+      config.middleware.use ExceptionNotification::Rack,
+                            email: {
+                              email_prefix: '[ERROR] ',
```

Comment:
Make this a setting as well. And make the default value `'[MarkUs Exception Notification] '`